cli/src/stats.rs

```diff
@@ -0,0 +1,53 @@
+use ansi_term::Colour::{Blue, Green, Red, Yellow};
+use clap::Parser;
+
+#[derive(Parser, Debug, Clone)]
+pub struct StatsCmd;
+
+impl StatsCmd {
+    pub fn execute(&self) -> Result<(), Box<dyn std::error::Error>> {
+        let pot_value = 1500;
+        let player_amounts = vec![3000, 2500, 4000, 3500];
+        let current_player = 2;
+
+        let poker_table = format!(
+            "{}\n\
+             {}\n\
+             {}\n\
+             {}\n\
+             {} {} {} {} {}\n\
+             {}\n\
+             {}\n\
+             {}\n\
+             {}\n\
+             {}\n\
+             {}\n\
+             {}\n\
+             {}\n\
+             {}",
+            Blue.bold().paint("+-----------------------------+"),
```

Comment:
We need to make the values dynamic here. Also this won't be affected by the screen resolutions right ?